packages/hdwallet-ledger-webusb/src/adapter.ts

```diff
@@ -60,15 +60,13 @@ export class WebUSBLedgerAdapter {
     }, APP_NAVIGATION_DELAY);
   }
 
-  public get(device: USBDevice): core.HDWallet | null {
-    if (!device.serialNumber) throw new Error("missing serial number");
-    return this.keyring.get(device.serialNumber);
+  public get(device: USBDevice): core.HDWallet {
+    return this.keyring.get((device as any).serialNumber);
```

Comment:
Ledgers do not have serial numbers

Reply:
If that's true, won't this crash or return undefined?

I'd rather just return a static value.

Reply:
Yeah....good catch. I was waiting for it to break but it didn't. On the axiom side we hardcode all ledgers to have deviceId aka serialNumber as "0001" Maybe we should hardcode it here too? @kaladinlight Do you have any ideas?

Reply:
The `0001` id is hardcoded on the ledger device itself. Whether it is the serialNumber or deviceID or both, I don't recall exactly. All I remember is there was no good unique identifiers at the time, so we just used the same one for everyone and said we weren't going to support multiple ledgers connected at once. Would have to query the device on the `window.navigator` to verify. My ledger doesn't even connect to ledger live anymore....

Reply:
Kevin is right. During pairing it was throwing the "missing serial number" error which broke the whole pairing flow, but I confirmed by removing that throw then after the pair step we are able to access device.serialNumber which is "0001"